docs/serving/configuration/deployment.md

```diff
@@ -106,3 +109,25 @@ data:
   # List of repositories for which tag to digest resolving should be skipped
   registries-skipping-tag-resolving: registry.example.com
 ```
+
+## Enable container-freezer service
+
+You can configure queue-proxy to call an endpoint when its traffic drops to zero or scales up from zero by modifying the `concurrency-state-endpoint` ConfigMap setting. With this configuration, you can achieve some features like freezing a running process in a user-container or billing based on the time it takes to process the request.
```

Comment:
On a second read-through, I think it might be clearer to readers to start off with what the freezer does and then get into how it does it. Something along the lines of this (though feel free to tweak as needed, it probably can be better than what I came up with):

```suggestion
You can configure queue-proxy to pause pods when not in use by enabling the `container-freezer` service. It calls a stand-alone service (via a user-specified endpoint) when a pod's traffic drops to zero or scales up from zero. To enable it, set `concurrency-state-endpoint` to a non-empty value. With this configuration, you can achieve some features like freezing a running process in a user-container or billing based on the time it takes to process the request.
```